main.tf

```
@@ -154,4 +155,15 @@ resource "kubernetes_secret" "secrets" {
   data = each.value
 }
 
+resource "google_compute_address" "ingress_external_ip" {
+  count = var.create_ingress_static_ip ? 1 : 0
+
+  name       = local.ingress_static_ip_name
+  region     = data.google_client_config.google_client.region
```

Remove empty line

&

Fix TF formatting.

```suggestion
  count  = var.create_static_ingress_ip ? 1 : 0
  name   = local.ingress_ip_name
  region = data.google_client_config.google_client.region
```